packages/wallets/jest.config.js

```diff
@@ -11,4 +11,5 @@ module.exports = {
         '\\.(ts|tsx)?$': '<rootDir>/jestTransformer.js',
         '^.+\\.jsx?$': 'babel-jest',
     },
+    transformIgnorePatterns: ['/node_modules/(?!(@deriv-com/ui)).+\\.js$'],
```

Comment:
hey @hasan-deriv 
seems like we already added @deriv-com/ui lib in `transformIgnorePatterns` in root jest config.
Do we really need it here?